firebase/database.rules.json

```diff
@@ -5,33 +5,38 @@
     "v2": {
       "announcement": {
         ".write": false,
-        ".read": true
+        ".read": "auth != null"
       },
       "projects": {
-        ".write": false,
-        ".read": "auth != null",
+        // project managers can read/write all attributes
+        ".write": "auth.token.projectManager === true",
+        ".read": "auth.token.projectManager === true",
         "$project_id": {
-          "status": {
-            ".write": "auth.token.projectManager === true"
+          "teamId": {
+            // only team members can read
+            ".read": "root.child('/v2/users/'+auth.uid+'/teamId').val() === data.val()"
```

Comment:
@laurentS this is the simple test to check if the given `teamId`of the project (`data.val()`) and the teamId of the user match.